File: backend/driver/src/main/java/dev/tpcoder/empfriendly/driver/model/OrderResponse.java
package dev.tpcoder.empfriendly.driver.model;

import dev.tpcoder.empfriendly.driver.enumeration.DriveType;
import java.time.LocalDateTime;

public record OrderResponse (
    String id,
    DriveType type,
    LocalDateTime createdDate

) {

}

File: backend/order/src/main/kotlin/dev/tpcoder/empfriendly/order/model/dto/OrderResponse.kt
package dev.tpcoder.empfriendly.order.model.dto

import dev.tpcoder.empfriendly.order.enumeration.DriveType
import java.time.LocalDateTime

data class OrderResponse(
    val id: String,
    val type: DriveType,
    val createdDate: LocalDateTime
)
